hack/ginkgo-e2e.sh

```diff
@@ -130,13 +128,12 @@ else
     "--auth_config=${AUTH_CONFIG:-}"
     "--cert_dir=${CERT_DIR:-}"
   )
-  host="https://${KUBE_MASTER_IP-}"
 fi
 
 # Use the kubectl binary from the same directory as the e2e binary.
 export PATH=$(dirname "${e2e}"):"${PATH}"
 "${e2e}" "${auth_config[@]:+${auth_config[@]}}" \
-  --host="$host" \
+  --host="https://${KUBE_MASTER_IP-}" \
```

Comment:
Yeah, it might make the code here a little clearer if --host were only provided when it was needed.
OTOH, that means duplicating exactly the same line 5 times, which also isn't great.
Maybe add a comment or something that mentions when --auth_config is used, --host is ignored?